### PR TITLE
add devise and User

### PR DESCRIPTION
**Pull request by megashurik**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+#  name                   :string
+#
+
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+    :recoverable, :rememberable, :trackable, :validatable
+
+  has_many :occupations
```
Comment (Fahrenhei7) [inline app/models/user.rb]: Remove associations from other project

File: app/views/users/confirmations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```
Comment (Fahrenhei7) [inline app/views/users/confirmations/new.html.erb]: convert all .erb to .slim (with something like erb2slim)

**Comment by Freika** [comment]
`newbranch` is a very bad name
Why no description in PR?

File: spec/factories/users.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :user do
+
```
Comment (Fahrenhei7) [inline spec/factories/users.rb]: fill factory. [example](https://gist.github.com/Fahrenhei7/e6bbdc306d69006c2f040d5f5d2456ce)

File: spec/models/user_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (Fahrenhei7) [inline spec/models/user_spec.rb]: add name validation tests with shoulda-matchers [documentation](https://github.com/thoughtbot/shoulda-matchers)

**Comment by Fahrenhei7** [comment]
Add acceptance tests for user login/register/logout (features tests)

File: app/models/user.rb (hunk 1 above)
Comment (megashurik) [inline app/models/user.rb]: done

File: app/views/users/confirmations/new.html.erb (hunk 2 above)
Comment (megashurik) [inline app/views/users/confirmations/new.html.erb]: done

**Pull request closed without merging**